runtime/compiler/optimizer/InterpreterEmulator.cpp

```diff
@@ -604,6 +607,38 @@ InterpreterEmulator::visitInvokedynamic()
    TR::ResolvedMethodSymbol *resolvedSymbol = 0;
    Operand *result = NULL;
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+   int32_t callSiteIndex = next2Bytes();
+   TR_ResolvedMethod * owningMethod = _methodSymbol->getResolvedMethod();
+
+   J9InvokeCacheEntry *invokeCache = (J9InvokeCacheEntry *) owningMethod->callSiteTableEntryAddress(callSiteIndex);
+   if (!invokeCache) return; // unresolved
```

Comment:
`invokeCache` will never be null, it's just a pointer into the side table. We should use `isUnresolvedCallSiteTableEntry`